_data/projects.yml

```diff
@@ -35,6 +35,14 @@
   contact: Bryan Harter, Michael Liu, David Stansby, Michael Hirsch
   keywords: ["magnetosphere","ionosphere_thermosphere_mesosphere","specific"]
 
+- name: CCSDSPy
+  code: https://github.com/ccsdspy/ccsdspy/
+  description: A Python package for reading CCSDS (spacecraft) packet data.
+  docs: "https://docs.ccsdspy.org/en/latest/#"
+  logo: https://docs.ccsdspy.org/en/latest/_static/logo.png
+  contact: Daniel da Silva
+  keywords: ["ccsds","specific"]
+
```

Comment:
This is missing the grading badges required to add a new project.

Reply:
Added the badges.